pkgs/development/compilers/llvm/10/compiler-rt.nix

```
@@ -41,6 +41,11 @@ stdenv.mkDerivation rec {
     "-DCMAKE_TRY_COMPILE_TARGET_TYPE=STATIC_LIBRARY"
   ] ++ stdenv.lib.optionals (bareMetal) [
     "-DCOMPILER_RT_OS_DIR=baremetal"
+  ] ++ stdenv.lib.optionals (stdenv.hostPlatform.isDarwin) [
```

```suggestion
  ] ++ lib.optionals (stdenv.hostPlatform.isDarwin) [
```

stdenv.lib is deprecated.